<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TV_remote</class>
 <widget class="QMainWindow" name="TV_remote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>571</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>264</width>
    <height>571</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>264</width>
    <height>571</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_volume_up">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>V+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_volume_down">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>V-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_channel_up">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>C+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_channel_down">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>C-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_power">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Power</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_mute">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Mute</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Cartoon_Network">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Peacock">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CNN">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ESPN">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HBO">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HGTV">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="History_Channel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MTV">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Nickelodeon">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>350</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_netflix">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>410</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Netflix</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_youtube">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>440</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Youtube</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_spotify">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>470</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Spotify</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Free_form">
    <property name="geometry">
     <rect>
      <x>119</x>
      <y>380</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_channel_return">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>380</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_channel_enter">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>380</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
   <widget class="QLabel" name="graphicsView_channel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>plain-black-background-02fh7564l8qq4m6d.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>264</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
